website/README.md

```diff
@@ -351,7 +431,7 @@ One more example - let's say that content is being moved to an external website.
 }
 ```
 
-If we no longer want the link to be in the side nav, we can simply remove it. If we do still want the link in the side nav, but pointing to an external destination, we need to slightly change the structure as such:
+If we no longer want the link to be in the side nav, we can simply remove it. If we do still want the link in the side nav, but pointing to an external destnation, we need to slightly change the structure as such:
```

Comment:
Idk if this one is already covered but just in case

Reply:
✅ added to https://github.com/hashicorp/nextjs-scripts/pull/161